localization/js/timer-text.js

```diff
@@ -0,0 +1,34 @@
+import {Component, TextComponent} from '@wonderlandengine/api';
+
+/**
+ * timer
+ */
+export class TimerText extends Component {
+    static TypeName = 'timer-text';
+
+    init() {
+        this.timer = 0.0;
+        this.seconds = 0;
+        this.engine.i18n.onLanguageChanged.add(this.onLanguageChanged.bind(this));
+
+        this.text = this.object.getComponent(TextComponent);
+    }
+
+    update(dt) {
+        this.timer += dt;
+        if(this.timer < 1.0) {
+            return;
+        }
+
+        this.timer -= 1.0;
+        ++this.seconds;
```

Comment:
```suggestion
        this.time += dt;
        this.seconds = Math.floor(this.time);
```

You will need to also rename the timer in init()

Reply:
The idea was to only update the text once per second though, so the if would need to stay (just in a differently)